xdcs-server/src/main/java/pl/edu/agh/xdcs/db/entity/HistoricalTaskEntity.java

```diff
@@ -39,6 +40,18 @@
     @Enumerated(EnumType.STRING)
     private Result result;
 
+    @Column(name = "KERNEL_ARGS_")
+    @ElementCollection(fetch = FetchType.EAGER)
+    private List<String> kernelArguments;
```

Comment:
Jak pominiesz tabelę XDCS_OR_REF_ to możesz stracić obiekt

Reply:
dodałem, ale nie wiem jak zrobić to bez tabeli łącznikowej, bo domyślnie robi się syf: tabela łącznikowa oraz kolumna w ObjectRefEntity, a docelowo powinien być brak dodatkowej kolumny w ObjectRefEntity oraz lista id-ków w kolumnie kernelArguments w HistoricalTaskEntity

Reply:
jest ok